hadoop-ozone/dist/src/main/smoketest/security/ozone-secure-token.robot

```diff
@@ -36,7 +36,7 @@ Get and use Token in Secure Cluster
     Run Keyword                  Kinit test user     testuser  testuser.keytab
 
 Get Token in Unsecure Cluster
-    ${output} =                  Execute             ozone sh token get
+    ${output} =                  Execute             ozone sh token get -t /tmp/ozone.token
```

Comment:
use a path that local to the invocation `./` rather than `/tmp`

Reply:
Made the changes, thanks!